tests/integration/records/multi_db/test_records_table2table.py

```diff
@@ -89,34 +105,18 @@ def move_and_verify(self, source_dbname: str, target_dbname: str) -> None:
         validator.validate(schema_name=target_schema_name,
                            table_name=TARGET_TABLE_NAME)
 
-        quoted_target = quote_schema_and_table(target_engine, target_schema_name, TARGET_TABLE_NAME)
+        quoted_target = quote_schema_and_table(None, target_schema_name, TARGET_TABLE_NAME,
+                                               db_engine=target_engine)
         sql = f"DROP TABLE {quoted_target}"
-        target_engine.execute(sql)
+        target_conn.exec_driver_sql(sql)  # type: ignore[attr-defined]
 
-        records_database_fixture.tear_down()
+        # records_database_fixture.tear_down()
+        source_conn.close()
```

Comment:
should these be using `with` instead to close the connection? if the test fails before it gets to closing the connection are we gonna be leaving hannging connections around?

Reply:
I'm going to turn the records_database_fixture.tear_down() back on, I don't know why I turned this off but that definitely contains closure of the connection.

Reply:
Used with syntax, brought back the tear_down